alacritty.yml

```diff
@@ -78,6 +78,12 @@
   # Set this to `None` to use the default theme variant.
   #gtk_theme_variant: None
 
+  # Disable vsync (changes require restart)
+  #
+  # Force to disable vsync, what can improve the input latency but can also add tearing.
+  # On Wayland, vsync is always disabled, setting this option has no effect.
```

Comment:
We should state that this will decrease performance significantly. We're talking like ~35% in some basic benchmarks.

Reply:
Done.